extensions/api/data-management/policydefinition/build.gradle.kts

```diff
@@ -25,6 +25,9 @@ dependencies {
     implementation(project(":common:util"))
     implementation(project(":extensions:api:api-core"))
     implementation(project(":extensions:api:data-management:api-configuration"))
+    implementation(project(":spi:policy-spi"))
+    implementation(project(":spi:contract-spi"))
+    implementation(project(":extensions:transaction:transaction-spi"))
 
     implementation("jakarta.ws.rs:jakarta.ws.rs-api:${rsApi}")
     testImplementation(testFixtures(project(":launchers:junit")))
```

Comment:
the following deps are not needed. pls remove:
```kotlin
    testImplementation(testFixtures(project(":launchers:junit")))
    testImplementation(project(":extensions:in-memory:contractdefinition-store-memory"))
```

Reply:
Hi Paul, without the junit I cannot import org.eclipse.dataspaceconnector.junit.launcher.EdcExtension, that I use in my IntegrationTest.

Without the contractDefinition-store-memory I cannot check if a Policy is already part of a contract definition in order to be able to delete it.

Reply:
Mmh, i see. Leave it in then. Strange thing is, that i deleted those lines, re-ran the test an it worked? That's why I added this comment in the first place. Maybe it was a caching issue. 
Alright, disregard the comment then.